charts/ocis/values.yaml

```diff
@@ -613,6 +613,15 @@ secretRefs:
   # -- The secret to secure the global notifications endpoint. Only system admins and users knowing that secret can call the global notifications POST/DELETE endpoints.
   globalNotificationsSecretRef: ""
 
+# Custom CA chain
+# If a custom CA chain is needed for the communcation with other services like S3 or notification services.
+# The secret needs to be manually created.
+customCAChain:
+  # -- Custom CA enables SSL_CERT_DIR in pods with the additional path /etc/ssl/custom.
+  enabled: false
+  # -- If custom CA chain is enabled this attribute mounts the existing secret to /etc/ssl/custom.
+  existingSecret: ""
```

Comment:
Is there a particular reason for the CA chain to be put in a secret? From what I know, a CA chain is public information.

Reply:
I changed it to a ConfigMap in https://github.com/owncloud/ocis-charts/pull/599/commits/e3443826ee3659970b262e7c767d2cac786a4c8d